tests/platform_tests/api/test_module.py

```diff
@@ -437,6 +455,8 @@ def test_reboot(self, duthosts, enum_rand_one_per_hwsku_hostname, localhost, pla
         reboot_timeout = 300
         for mod_idx in range(self.num_modules):
             mod_name = module.get_name(platform_api_conn, mod_idx)
+            if self.skip_module_other_than_myself(mod_idx,platform_api_conn):
+                continue
             if mod_name in self.skip_mod_list:
                 logger.info("skipping reboot for module {} ".format(mod_name))
             else:
```

Comment:
test_reboot still fails. Need an additoinal change to check as below. module_reboot is coming as a boolean
```
module_reboot is True
```

Reply:
Yes - we noticed that in our runs. Will push the fix in this PR itself.

Reply:
@judyjoseph, Added fix for the 'module_reboot' parameter to this PR itself. Please check..